conditional/templates/dashboard.html

```diff
@@ -17,7 +17,7 @@ <h5 class="email">{{username}}@csh.rit.edu</h5>
                         {% if active %}
                         <span class="label label-success">Active</span> {% else %}
                         <span class="label label-danger">Inactive</span> {% endif %} {% if onfloor %}
-                        <span class="label label-primary">On-floor</span> {% else %}
+                        <span class="label label-primary">On-floor Status</span> {% else %}
                         <span class="label label-default">Off-floor</span> {% endif %} {% if voting %}
```

Comment:
Should this also be changed to `Off-floor status`?

Reply:
I had that thought too, but since you can't live on floor and have off-floor status like you can the other way around (without some j-board fuckery) I didn't know if I should change it

Reply:
I think it makes sense to change it just to keep symmetry between the two. It's still reflecting what your status is, not where you're living.